src/lib/Zikula/Bundle/CoreInstallerBundle/Stage/LocaleStage.php

```diff
@@ -71,7 +71,7 @@ public function isNecessary()
             if (($locale !== false) && (in_array($locale, $installedLanguages))) {
                 $this->writeParams(array('locale' => $locale));
 
-                return false;
+                return true;
```

Comment:
I'm not sure this is correct. The method should return `false` is the method is **not** necessary. The code immediately before this change set the locale based on browser detection and so it would seem that the stage is **not** necessary because the task of setting the locale is **complete**. returning `true` will display the locale selector, which seems not necessary if it has been detected from the browser... ?